ddtrace/contrib/pytest/plugin.py

```diff
@@ -161,11 +165,6 @@ def pytest_runtest_makereport(item, call):
     if span is None:
         return
 
-    called_without_status = call.when == "call" and span.get_tag(test.STATUS) is None
```

Comment:
we can't do this check anymore because we set `test.STATUS` before. I think it's useless anyway (I might be wrong). This is in place (if I understand it correctly) to avoid running the code below more than once, if there have been no errors in `setup`. But there could be errors in `teardown`. The logic is convoluted too, so I think we gain by removing it